lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -280,11 +316,12 @@ def setup_access_token
         response['access_token']
       end
 
-      def commit(action, post, authorization = nil)
+      def commit(action, post, authorization = nil, method = :post)
         begin
-          raw_response = (action == :verify_payment ? ssl_get("#{base_url}/payments/#{post}", headers) : ssl_post(url(post, action, authorization), post.to_json, headers))
+          raw_response = ssl_request(method, url(action, authorization), post.to_json, headers(action))
```

Comment:
I see there's a path where `post` can be `nil`. Does passing in `nil.to_json` have any issues?

Reply:
I think because we initialize the `post` variable within each of the methods that eventually calls `commit`, the value of `post` should always at least be an empty hash. We could conceivably default the value to `{}`, but I think this is a common pattern in AM.